aiida_vasp/parsers/vasp.py

```diff
@@ -17,78 +115,166 @@ class VaspParser(BaseParser):
 
     def __init__(self, calc):
         super(VaspParser, self).__init__(calc)
+
         self.out_folder = None
-        self.vrp = None
-        self.dcp = None
+        self._settings = DEFAULT_OPTIONS
+
+        try:
+            self._settings.update(self._calc.inp.settings.get_dict()['parser_settings'])
```

Comment:
replace the line `self._settings = DEFAULT_OPTIONS` and this whole try-except with
```
self._settings = self._calc.inp.settings.get_dict().get('parser_settings', DEFAULT_OPTIONS)
```